guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -57,6 +76,29 @@ The `filename` option contains the full path to the file that downloads and exec
 |Grub2 UEFI HTTP | `http://_{smartproxy-example-com}_:8000/httpboot/grub2/grubx64.efi` | Requires the `httpboot` feature and renders the `filename` as a full URL where _{smartproxy-example-com}_ is a known host name of {SmartProxy} in {Project}.
 |=======
 
+=== Troubleshooting DHCP Problems in {Project}
+
+{Project} can manage an ISC DHCP server on internal or external DHCP {SmartProxy}. {Project} can list, create, and delete DHCP reservations and leases. However, there are a number of problems that you might encounter on occasions.
+
+.PXE loader option change
+
+When the PXE loader option is changed for an existing host, this causes a DHCP conflict. This is a known issue. Until this issue is fixed, the only workaround is to overwrite the DHCP entry.
+
+.Incorrect permissions on DHCP files
+
+An operating system update can reset the `dhcpd` package. This causes the permissions of important directories and files to reset so that the DHCP {SmartProxy} cannot read required information.
+For more information, see https://access.redhat.com/solutions/2952061[DHCP error while provisioning host from Satellite server Error ERF12-6899 ProxyAPI::ProxyException: Unable to set DHCP entry RestClient::ResourceNotFound 404 Resource Not Found] on Red{nbsp}Hat Knowledgebase.
+
+.Changing the DHCP {SmartProxy} entry
+
+{Project} manages DHCP records only for hosts that are assigned to subnets with a DHCP {SmartProxy} set. If you create a host and then clear or change the DHCP {SmartProxy}, when you attempt to delete the host, the action fails.
+
+If you create a host without setting the DHCP {SmartProxy} and then try to set the DHCP {SmartProxy}, this causes DHCP conflicts.
+
+.Deleted hosts entries in the dhcpd.leases file
+
+Any changes to a DHCP lease are appended to the end of the `dhcpd.leases` file. Because entries are appended to the file, it is possible that two or more entries of the same lease can exist in the `dhcpd.leases` file at the same time. When there are two or more entries of the same lease, the last entry in the file takes precedence. Group, subgroup and host declarations in the lease file are processed in the same manner. If a lease is deleted, `{ deleted; }` is appended to the declaration.
+
```

Comment:
I'd be fine putting this into a KBASE, I am actually surprised you are letting this content in ;-) Cool.

Reply:
shhhh